src/upgrades/v0_9_0/ca_objects_migration.rs

```diff
@@ -267,32 +261,45 @@ impl UpgradeStore for CasStoreMigration {
                     );
                 }
 
-                trace!("  command: {}", cmd_key);
-                let mut old_cmd: OldStoredCaCommand = self.get(&cmd_key)?;
+                // Read and parse the old command.
+                let mut old_cmd: OldStoredCaCommand = self.get(&old_cmd_key)?;
 
+                // If the command was a success, then it will have events. Successful commands
+                // that resulted in no changes are simply not recorded. That said, it may turn
+                // out that the event list to migrate is empty - in that case we skip this command
+                // and continue the command loop.
+                //
+                // Note that if the command resulted in an error we do not not get 'Some' empty
+                // vec of events, but we get a None. So such commands will be migrated.
```

Comment:
will NOT be migrated? there's no else block handling the None case.

Reply:
hopefully more clear now..

                // Note that if the command resulted in an error we do not not get 'Some' empty
                // vec of events, but we get a None. So we skip this block. Error commands will
                // be migrated - we won't call continue as they are expected to have no events.